market/connector/google-geo-api/meta.json

```diff
@@ -0,0 +1,62 @@
+{
+	"id": "google-geo-api",
+	"name": "Google Map API",
+	"tags": ["connector", "map"],
+	"installers": [
+		{
+			"id": "project-resource",
+			"data": {
+				"dialog":
+				{
+					"source": "google-geo-api-demo/src_hd",
+					"ids": [ 
+						"google.getp.map"
+					]
+				},
+				"resources": [
+					{
+						"source": "google-geo-api-demo/dataclasses/google/getp/Data.ivyClass",
+						"path": "dataclasses/google/getp/Data.ivyClass"
+					},
+					{
+						"source": "google-geo-api-demo/processes/searchLocation.mod",
+						"path": "processes/searchLocation.mod"
+					},
+					{
+						"source": "google-geo-api-demo/webContent/layouts/js/jquery-3.6.0.min.js",
+						"path": "webContent/layouts/js/jquery-3.6.0.min.js"
+					},
+					{
+						"source": "google-geo-api-demo/webContent/layouts/js/map-dialog-handler.js",
+						"path": "webContent/layouts/js/map-dialog-handler.js"
+					},
+					{
+						"source": "google-geo-api-demo/webContent/layouts/images/axon_ivy_logo.svg",
```

Comment:
I think you can omit the installation of logos, layouts and the like. They will be automatically created by the 'project-resource/dialog' installer. The only exception is, if you have modified them ... but then I'd expect these modifications to reside in other files than the default layout aritfacts.

- [ ] copy only artifacts that you have created/contributed

Reply:
- [x] copy only artifacts that you have created/contributed

Reply:
I think there are still some left-overs in the meta.json that can be removed

- [ ] don't copy 'frame-8.xhtml' and 'ivy-common-8.css'

 ![dontCopyStandardHtmlParts](https://user-images.githubusercontent.com/15085693/116097911-e87ba300-a6aa-11eb-8e28-6e65b2ab7746.png)

Reply:
- [x] don't copy 'frame-8.xhtml' and 'ivy-common-8.css'